projects/go-lib/src/lib/components/go-side-nav/go-side-nav/go-side-nav.component.html

```diff
@@ -21,9 +21,10 @@
                      *ngIf="navAppDrawer?.appDrawerConfig"
                      #appDrawerActionSheet>
       <ng-container go-action-sheet__button>
-        <go-icon class="go-side-nav__app-drawer--icon"
+        <go-icon-button class="go-side-nav__app-drawer--icon"
                  [ngClass]="{'go-side-nav__app-drawer--icon-collapsed': !navService.navOpen}"
-                 [icon]="navService.navOpen ? 'more_vert' : navAppDrawer.currentAppIcon"></go-icon>
+                 buttonSize="large"
+                 [buttonIcon]="navService.navOpen ? 'more_vert' : navAppDrawer.currentAppIcon"></go-icon-button>
```

Comment:
Would you mind cleaning up the formatting here a bit, seeing as how you're already editing this code?
```html
<go-icon-button
  buttonSize="large"
  class="go-side-nav__app-drawer--icon"
  [ngClass]="{'go-side-nav__app-drawer--icon-collapsed': !navService.navOpen}"
  [buttonIcon]="navService.navOpen ? 'more_vert' : navAppDrawer.currentAppIcon">
</go-icon-button
```

Reply:
Done.
I hope I did justice to the formatting (please check the latest commit)!
And if time and VSC allows, we can work on creating our own custom settings for prettier.

Reply:
> And if time and VSC allows, we can work on creating our own custom settings for prettier.

That would be cool